src/phoenix/experimental/evals/models/vertexai.py

```diff
@@ -11,8 +11,8 @@
 except ImportError:
     MINIMUM_VERTEX_AI_VERSION = "1.33.0"
     raise ImportError(
-        "Could not import necessary dependencies to use VertexAI. "
-        "Please install them with `pip install google-cloud-aiplatform>={MINIMUM_VERTEXAI_VERSION}`."
+        "Could not import necessary dependencies to use VertexAI. Please install them with"
+        f"`pip install google-cloud-aiplatform>={MINIMUM_VERTEX_AI_VERSION}`."
```

Comment:
This import strategy requires that Vertex AI is installed, even if someone is only using `OpenAIModel`. Similarly, it requires that `openai` is installed, even if someone only wants to use `VertexAIModel`.

To reproduce, install `arize-phoenix` and `openai` in a new environment and run

```python
from phoenix.experimental.evals.models import OpenAIModel
```

This results in a Vertex AI import error.

We need to dynamically import the modules inside the class itself to contain the dependency.

Reply:
You are correct, it has been modified to prevent this in the next commit